taskcluster/ci/toolchain/android.yml

```diff
@@ -56,5 +57,10 @@ linux64-android-gradle-dependencies:
         env:
             # TODO do no hardcode
             ANDROID_SDK_ROOT: /builds/worker/fetches/android-sdk-linux
-        max-run-time: 4000
+        max-run-time: 14400
```

Comment:
4 hours!? I know this is long-running, but does it need to be that long?

Reply:
I agree we should revisit this. I just ported the value we've had in focus: 

https://github.com/mozilla-mobile/firefox-android/pull/279/commits/6f3b50c9d083e35df53ad3c4018c159d12137e5f#diff-efa79df56d923fbdbf87bf32aed331a202d755ce7e82480710dcc5db01551ee0L57